src/njoy21/input/argument/parser/test/parser.test.cpp

```diff
@@ -0,0 +1,3 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
```

Comment:
See the [directory](https://github.com/njoy/NJOY21/tree/feature/TAB1/src/njoy21/input/argument/parser/test)